pkg/scheduler/metrics/metrics.go

```diff
@@ -23,90 +23,90 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-const schedulerSubsystem = "scheduler"
+const (
+	SchedulerSubsystem = "scheduler"
+	SchedulingLatencyName = "scheduling_latencies_summary"
```

Comment:
I don't think it's too useful to have this const. Maybe just put the string directly inside the metric definition (like other metrics)?

Reply:
Why do we want to have operation types as a consts and and metric name as a string? Shouldn't we have coherent style - all strings or all consts?

Reply:
Yes, you have a valid point there.

So ideally we should make all hard-coded strings in files as global constants (so we only define the 'real' string once and later on only have references to it). However, moving too many things to global consts makes the file larger and harder to read. So the rule I'm usually following is to make a string const only if it's referenced enough times (3-4) in the codebase, so it's worth it. But this is subjective :)